vulnerabilities/importers/ruby.py

```diff
@@ -1,76 +1,126 @@
-import urllib.request
-from urllib.error import HTTPError
-from zipfile import ZipFile
-from io import BytesIO
-import yaml
+from json import JSONDecodeError
```

Comment:
Unrelated, but the license/copyright header is missing in this file. Mind adding it?

Reply:
Done